login.go

```diff
@@ -39,7 +40,7 @@ func (c *LoginController) Authorize(ctx *app.AuthorizeLoginContext) error {
 
 // Generate runs the authorize action.
 func (c *LoginController) Generate(ctx *app.GenerateLoginContext) error {
-	if !Development {
+	if !viper.GetBool("developer.mode.enabled") {
```

Comment:
So you're suggesting this?

```go
package configuration
func isDeveloperModeEnabled() bool {
    return viper.GetBool("developer.mode.enabled")
}
```

And you want the same for things like the postgres port/user/password and so forth?

I don't see any benefit for this. Even if we distribute viper calls all over the place and decide to rename things in the config file we can still use aliases in viper to reference the old keys.

Besides, if we're using viper under the hood, I don't see a particular reasons why we want to encapsulate it?